lua/autorun/client/cl_ross_derma.lua

```diff
@@ -0,0 +1,152 @@
+    self.cSpeed = !colorSpeed && 7 * 100 || colorSpeed * 700;
+    self.cCopy = self.dColor // color copy to decrement
+
+    self.Paint = function(s, w, h)
+        if !CLIENT then return end;
+        if !self.initedHover then return end;
+
+        local incTo = self.IncTo; // Increment color to;
+        local cCopy = self.cCopy;
+        local dis = self.Disable
+        local hov = self:IsHovered()
+        
+        if dis then
+            draw.RoundedBox(roundBorder, 0, 0, w, h, Color(cCopy.r, cCopy.g, cCopy.b, 100))
+            return;
+        end
+        local red, green, blue = self.dColor.r, self.dColor.g, self.dColor.b
+        self.dColor = {
+            r = appr(red, hov && incTo.r || cCopy.r, FrameTime() * self.cSpeed),
+            g = appr(green, hov && incTo.g || cCopy.g, FrameTime() * self.cSpeed),
+            b = appr(blue, hov && incTo.b || cCopy.b, FrameTime() * self.cSpeed)
+        }
+        draw.RoundedBox(roundBorder, 0, 0, w, h, self.dColor)
+
+        if self.bColor then
+            surface.SetDrawColor(Color(40, 40, 40))
+            surface.DrawOutlinedRect( 0, 0, w, h, 1 )
+        end;
+    end;
+end;
+
+// I've taken it from the helix;
+function BetterWrapText(text, maxWidth, font)
+    font = font or "DefaultFont"
+    surface.SetFont(font)
+
+    local words = string.Explode("%s", text, true)
+    local lines = {}
+    local line = ""
+    local lineWidth = 0
+
+    -- we don't need to calculate wrapping if we're under the max width
+    if (surface.GetTextSize(text) <= maxWidth) then
+        return {text}
+    end
+
+    for i = 1, #words do
+        local word = words[i]
+        local wordWidth = surface.GetTextSize(word)
+
+        -- this word is very long so we have to split it by character
+        if (wordWidth > maxWidth) then
+            local newWidth
+
+            for i2 = 1, string.len(word) do
+                local character = word[i2]
+                newWidth = surface.GetTextSize(line .. character)
+
+                -- if current line + next character is too wide, we'll shove the next character onto the next line
+                if (newWidth > maxWidth) then
+                    lines[#lines + 1] = line
+                    line = ""
+                end
+
+                line = line .. character
+            end
+
+            lineWidth = newWidth
+            continue
+        end
+
+        local newLine = line .. " " .. word
+        local newWidth = surface.GetTextSize(newLine)
+
+        if (newWidth > maxWidth) then
+            -- adding this word will bring us over the max width
+            lines[#lines + 1] = line
+
+            line = word
+            lineWidth = wordWidth
+        else
+            -- otherwise we tack on the new word and continue
+            line = newLine
+            lineWidth = newWidth
+        end
+    end
+
+    if (line != "") then
+        lines[#lines + 1] = line
+    end
+
+    return lines
+end
+    
+function panelMeta:Close()
```

Comment:
i would rather not put this in, it might cause issues and its totally out of scope of this change. 

(meaning, this whole file, should be removed, and i am just commenting here as an example of something that seems like a potential glitch)

Reply:
> i would rather not put this in, it might cause issues and its totally out of scope of this change.
> 
> (meaning, this whole file, should be removed, and i am just commenting here as an example of something that seems like a potential glitch)

Agreed. I left this file for future purposes if I'll make a new VGUIs in future.